mapic/mistapiconnector_app.go

```diff
@@ -506,18 +525,66 @@ func (mc *mac) reconcileLoop(ctx context.Context) {
 		case <-ctx.Done():
 			return
 		case <-ticker.C:
-		case <-mc.multistreamUpdated:
+		case <-mc.streamUpdated:
 		}
 		mistState, err := mc.mist.GetState()
 		if err != nil {
 			glog.Errorf("error executing query on Mist, cannot reconcile err=%v", err)
 			continue
 		}
+		mc.reconcileStreams(mistState)
 		mc.reconcileMultistream(mistState)
 		mc.processStats(mistState)
 	}
 }
 
+func (mc *mac) reconcileStreams(mistState clients.MistState) {
+	for streamName, _ := range mistState.ActiveStreams {
+		// read map directly to avoid creating streamInfo object even for playback-only (pull) streams
```

Comment:
I'm not sure I understand this comment. Could you maybe rephrase? We don't create any stream info here 🤔

Reply:
This reads from the map instead of calling `getStreamInfo`, which would fetch the info object even for the playback streams. I can rephrase it

Reply:
Removed the comment altogether with your suggested change.